plugins/plugin-site/src/components/ReportAProblem.jsx

```diff
@@ -3,10 +3,22 @@ import PropTypes from 'prop-types';
 
 function ReportAProblem({reportProblemTitle, reportProblemUrl, reportProblemRelativeSourcePath}) {
     const title = `Report a problem with ${reportProblemRelativeSourcePath}`;
-    const pluginSiteReportUrl = `https://github.com/jenkins-infra/plugin-site/issues/new?labels=bug&template=4-bug.md&title=${reportProblemTitle} page - TODO: Put a summary here&body=Problem with the [${reportProblemTitle}](https://plugins.jenkins.io${reportProblemUrl}) page, [source file](https://github.com/jenkins-infra/plugin-site/tree/master/src/${reportProblemRelativeSourcePath})%0A%0ATODO: Describe the expected and actual behavior here %0A%0A%23%23 Screenshots %0A%0A TODO: Add screenshots if possible %0A%0A%23%23 Possible Solution %0A%0A%3C!-- If you have suggestions on a fix for the bug, please describe it here. --%3E %0A%0AN/A`;
-    return (
+    const body = `Problem with the [${reportProblemTitle}](https://plugins.jenkins.io${reportProblemUrl}) page, [source file](https://github.com/jenkins-infra/plugin-site/tree/master/src/${reportProblemRelativeSourcePath})
+
+    TODO: Describe the expected and actual behavior here
+
+    %23%23 Screenshots
+    TODO: Add screenshots if possible
+
+    %23%23 Possible Solution
+
+    %3C!-- If you have suggestions on a fix for the bug, please describe it here. --%3E
+
+    N/A`.replace(/\n */g, '%0A');
```

Comment:
I also like the splitting into many lines.

Did you actually change anything here? or just add new lines?

I'm trying to turn these into reusable components - https://github.com/halkeye/jenkins-io-react/blob/main/src/components/ReportAProblem.jsx#L11 (eventually)

Reply:
Just added newlines. The URLSerchParams implementation looks nice!